app/controllers/wires_controller.rb

```diff
@@ -71,6 +71,19 @@ def send_wire
 
     @wire.send_wire!
 
+    if params[:charge_fee] == "1"
+      disbursement = DisbursementService::Create.new(
+        name: "Low-value wire transfer fee",
```

Comment:
Can we call it Under $500 wire...

Reply:
hmmmm... having an amount in the memo might be weird